cg/server/api.py

```diff
@@ -489,7 +489,7 @@ def parse_orderform():
         error_message = error.message if hasattr(error, "message") else str(error)
         http_error_response = http.HTTPStatus.INTERNAL_SERVER_ERROR
     else:
-        return jsonify(**parsed_order.dict())
+        return jsonify(**parsed_order.model_dump())
```

Comment:
What does jsonify do? Does it do the same as `return parsed_order.model_dump_json()`?

Reply:
What is jsonify?

Reply:
Oh, did not even notice it. Hm, no idea. Reading docs...
```
This function wraps dumps to add a few enhancements that make
life easier. It turns the JSON output into a ~flask.Response
```

Reply:
So I don't think it returns the same as `model_dump_json()`

Reply:
Oh, not the same then! model_dump_json() gives a string I think.

Reply:
Or maybe? All examples seems to give a string formatted as the class...

Reply:
Might be the case! Going to leave it for now, seems a little out of scope